### PR TITLE
Hot Fixes

### PR DESCRIPTION
**Pull request by mattbarker016**
- Changed response for routing `data` from `JSON.stringify(possibleRoutes)`to just `possibleRoutes`.
- Added `*graph-cache*` to `.gitignore`

### CODE REVIEW

File: src/RouteRouter.js [hunk 1]
```diff
@@ -141,8 +141,9 @@ class RouteRouter extends AbstractRouter {
                 })
 
             }
-            //res.set('Content-Type', 'application/json');
-            return JSON.stringify(possibleRoutes);
+            // res.set('Content-Type', 'application/json');
+            // return JSON.stringify(possibleRoutes);
+            return possibleRoutes
```
Comment (AAAstorga) [inline src/RouteRouter.js]: If you don't want it stringified, please do it on all Routers. Thanks

Comment (AAAstorga) [reply src/RouteRouter.js]: Also please include an explanation on why this change is happening

Comment (mattbarker016) [reply src/RouteRouter.js]: The value for the data key was an array made into a string. You can return the array object, which makes it easier to iterate over the unwrapped data on the client-side.

**Comment by AAAstorga** [comment]
Looks good. Please merge into unstable branch

**Pull request merged**